src/main/java/org/commcare/formplayer/services/SubmitService.java

```diff
@@ -27,11 +27,11 @@ public class SubmitService {
 
     private CategoryTimingHelper.RecordingTimer submitTimer;
 
-    public String submitForm(String formXml, String submitUrl) {
+    public String submitForm(Object body, String submitUrl) {
```

Comment:
Should this not be `MultiValueMap`? I don't see this function used anywhere else.

Reply:
https://github.com/dimagi/formplayer/pull/1209/commits/23a5881c68c9c4cb9b844c02506e6085caff0f41